style.css

```diff
@@ -5,8 +5,116 @@ html { box-sizing: border-box; }
 
 body {
     text-rendering: optimizeLegibility;
+    margin: 0px auto;
+    padding: 30px;
 }
 
+.zbozi {
+    overflow: hidden;
+    width: 670px;
+    margin: 20px auto;
+    border-radius: 30px;
+}
+
+.vizual {
+    width: 50%;
+    float: left;
+    height: 440px;
+    background: linear-gradient(#06d7c1, #50cc49) ;
+    padding: 30px;
+    clear: both;
+}
+
+.detaily {
+    width: 50%;
+    float: left;
+    height: 440px;
+    padding: 30px;
+}
+
+.cenovka {
+    text-align: right;
+    position: relative;
+    bottom: -200px;
+}
+
+.nazev {
+    font-size: 20px;
+    font-weight: 500;
+}
+
+.produkt {
+    color: darkgrey;
+}
+
+.slogan {
+    color: white;
+    font-size: 16px;
+    font-weight: 500px;
+}
+
+.cena {
+    color: white;
+    font-size: 28px;
+    font-weight: 500;
+}
+
+.zaruka {
+    color: white;
+    font-size: 12px;
+    font-weight: 400px;
+}
+
+
+.velikosti {
+    display: flex;
+    justify-content: space-between;
+    padding-inline-start: 0;
+}
+
+.vybrana {
+    border-color: #f15f42;
+    border-width: 1px;
+    border-style: solid;
+    border-radius: 30px;
+    padding: 5px;
+    color: #f15f42;
+}
+
+.velikosti li {
+    display: inline;
+    align-self: center;
+    position: relative;
+    list-style-type: none;
+}
+
+.bota {
+    position: absolute; 
+    top: 180px;
+    left: 430px;
```

Comment:
Tady by se více hodilo nepozicovat to celkově odvrchu a zprava (při různých velikostech obrazovky se ti to dost rozhazuje...), ale radši pozicovat ze spodu a zleva (left: -80px; bottom: 0;) chceš totiž hlavně to, aby se bota posunula o kousek doleva oproti základu. (Aby se to ale uplatňovalo ne k celému dokumentu, bude to chtít na rodiči (kartičce zboží) mít např. position: relative...)